build/Dockerfile

```
@@ -331,3 +331,23 @@ USER cartesi
 
 # Set the Go supervisor as the command.
 CMD [ "cartesi-rollups-node" ]
+
+
+# STAGE: rollups-ci-base
+#
+# This stage prepares the base CI image
+FROM debian:bookworm as rollups-node-ci-base
```

```suggestion
FROM ${BASE_IMAGE} as rollups-node-ci-base
```

I would use the base image to be consistent.